src/navbar/navbar.tsx

```diff
@@ -225,7 +225,9 @@ const Component = <T,>(
                             />
                             <NavbarActionButtons
                                 actionButtons={
-                                    actionButtons && actionButtons.desktop
+                                    actionButtons &&
+                                    (actionButtons?.mobile ||
+                                        actionButtons.desktop)
```

Comment:
this doesn't work for desktop though. the component has to only take desktop buttons in desktop viewport

I suggest:

1. update `NavbarActionButtons` to take in both sets of action buttons i.e.
```
interface Props {
    actionButtons?: NavbarActionButtonsProps | undefined;
    ...
}
```
2. derive `uncollapsableActionButtons` and `collapsableActionButtons` from mobile, then desktop i.e.
```
const uncollapsableActionButtons =(actionButtons?.mobile ||
                                        actionButtons.desktop).filter(
            (actionButton) => !!actionButton.uncollapsible
        );`
```